go/hamming/hamming.go

```diff
@@ -1,4 +1,37 @@
+// This package implements an hamming distance calculation function
+// aimed at DNA strands
+
 package hamming
 
-func Distance(a, b string) (int, error) {
+import (
+	"errors"
+	"strings"
+)
+
+/*  Distance returns the hamming distance between two dna strands
+    passed as string parameters. Returns -1 if the strands are of
+	different lengths.
+    parameters :
+		a : strand as a string
+		b : strand as a string
+*/
+func Distance(a, b string) (distance int, distanceError error) {
```

Comment:
while in some cases the named return parameters are useful, I think I have seen them being used only once _in the wild_.  The problem is that they bring a bit of confusion with them (where do the names come from and why there are no variables in the return statement). I am not sure I really like the use here, but I am also not strongly against it...

Reply:
No worries, I'll remove them (as well as in #8 and #7 ^^^)